### DISCUSSION TITLE
Welcome to esp32-geiger-counter Discussions!

### DISCUSSION THREAD

**Discussion by grillbaer** [grillbaer/esp32-geiger-counter#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wery67564** [comment]
What is Pulse B to J1 for?

**Comment by grillbaer** [comment]
JP1 closed = buzzer on (click sound per pulse),
JP1 open = buzzer off (silent, no click sounds)
